src/libxrpl/protocol/Indexes.cpp

```diff
@@ -301,6 +304,27 @@ depositPreauth(AccountID const& owner, AccountID const& preauthorized) noexcept
         indexHash(LedgerNameSpace::DEPOSIT_PREAUTH, owner, preauthorized)};
 }
 
+Keylet
+depositPreauth(AccountID const& owner, STArray const& authCreds) noexcept
+{
+    std::string s;
+    std::vector<uint256> hashes;
+    hashes.reserve(authCreds.size());
+    for (auto const& o : authCreds)
+    {
+        hashes.push_back(sha512Half(
+            o.getAccountID(sfIssuer), o.getFieldVL(sfCredentialType)));
+    }
+
+    // eleminate duplicates
+    std::sort(hashes.begin(), hashes.end());
+    hashes.erase(std::unique(hashes.begin(), hashes.end()), hashes.end());
+
+    return {
+        ltDEPOSIT_PREAUTH,
```

Comment:
This needs to be updated to the credential version.

Reply:
fixed